### PR TITLE
Fix Kafka error connection on Travis CI

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,12 +36,21 @@ before_install:
   - sudo apt-get install jq
   - curl -s https://raw.githubusercontent.com/edenlabllc/ci-utils/umbrella_v2/init-redis.sh -o init-redis.sh; sudo sh ./init-redis.sh
   - curl -s https://raw.githubusercontent.com/edenlabllc/ci-utils/umbrella_v2/init-mongodb.sh -o init-mongodb.sh; sudo sh ./init-mongodb.sh
+  # install kafka
+  - wget http://www.us.apache.org/dist/kafka/2.0.0/kafka_2.12-2.0.0.tgz -O kafka.tgz
```
Comment (AlexKovalevych) [inline .travis.yml]: please move install kafka to `ci-utils` `umbrella_v2` branch, `init-kafka.sh`

File: bin/install-kafka.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (AlexKovalevych) [inline bin/install-kafka.sh]: not used?

Comment (PavelVesnin) [reply bin/install-kafka.sh]: not used

File: .travis.yml (hunk 1 above)
Comment (PavelVesnin) [reply .travis.yml]: done

**Pull request merged**